Operational-Procedures/Auxiliary-Telescope/LATISS-Master-Calibrations-Procedure/latiss-master-calibrations-procedure.rst

```diff
@@ -0,0 +1,212 @@
+  
+These options are constrained by the fact that one calibration depends on the existence of the previous one (i.e., to build a dark, a bias is necessary, and to build a flat, a dark and a bias are necessary).
+
+If desired, defects can be constructed from darks and flats (in which case  `script_mode` must be ``BIAS_DARK_FLAT``), and a PTC per detector per amplifier constructed form the flats. Note that the PTC assumes that a sequence of flat pairs has been taken, each pair taken at the same exposure time (in order to produce a PTC, `script_mode` must be ``BIAS_DARK_FLAT`` too).
+
+For more information about calibrations production (including verification and certification), please consult the `Constructing Calibrations documentation`_.
+
+.. _cp_verify: https://github.com/lsst/cp_verify
+.. _DMTN-101: https://dmtn-101.lsst.io/
+.. _Constructing Calibrations documentation: https://lsst.ncsa.illinois.edu/~czw/pipelines_lsst_io/_build/html/modules/lsst.cp.pipe/constructing-calibrations.html 
+
+.. _Latiss-Master-Calibrations-Procedure-Prerequisites:
+
+
+Prerequisites
+=============
+
+- You should be logged into the LSST Operations and Visualization Enviroment (LOVE) at the Summit :ref:`operational environment <Observing-Interface-Operational-Environments>`.
+- The script assumes (and checks) that the ``LATISS`` and ``OCPS`` components are all ``ENABLED``, and that the latter has been ``ENABLED`` with the configuration of ``LATISS``. The instrument and the ``OCPS`` can be enabled, for example, from a notebook: 
+
+.. code-block:: python
+
+    import asyncio
+    from lsst.ts import salobj
+    from lsst.ts.observatory.control.maintel.latiss import Latiss
+
+    do_enable_camera = True
+    do_enable_ocps = True
+
+    domain = salobj.Domain()
+    latiss = Latiss(domain)
+    ocps = salobj.Remote(domain, "OCPS")
+    await asyncio.gather(latiss.start_task, ocps.start_task)
+
+    if do_enable_camera:
+       await latiss.enable()
+
+    if do_enable_ocps:
+        instrument="LATISS"
+        ack = await ocps.cmd_start.set_start(settingsToApply=instrument)
+        if ack.ack != salobj.SalRetCode.CMD_COMPLETE:
+            ack.print_vars()
+
+        ack = await ocps.cmd_enable.set_start()
+        if ack.ack != salobj.SalRetCode.CMD_COMPLETE:
+            ack.print_vars()
+
+.. _butler: https://pipelines.lsst.io/v/daily/modules/lsst.daf.butler/index.html
+
+.. _Latiss-Master-Calibrations-Procedure-Post-Conditions:
+
+Post-Condition
+==============
+
+- A (daily) master calibration image per detector will be certified in a butler ``CALIBRATION`` `collection`_.
+
+.. _collection: https://pipelines.lsst.io/v/daily/modules/lsst.daf.butler/organizing.html
+
+.. _Latiss-Master-Calibrations-Procedure-Steps:
+
+Procedure Steps
+===============
+
+Once you are logged into LOVE, click on the ``MTQueue`` panel, as circled on the right side of the figure below (for completeness, ``ATQueue`` to launch ``LSSTComCam`` scripts is circled on the right):
```

Comment:
LATISS is in Auxiliary Telescope (AT) so it should use `ATQueue`... ComCam is Main Telescope (MT) so it should use `MTQueue`...